src/storage/src/hummock/sstable/block.rs

```diff
@@ -30,30 +30,30 @@ pub const DEFAULT_ENTRY_SIZE: usize = 16;
 
 pub struct Block {
     /// Uncompressed entries data.
-    data: Bytes,
+    data: Vec<u8>,
```

Comment:
I use vector because if some Bytes object is produced by `SSTableBuilder`, its capacity may be much larger than length. Hold a slice reference is not a good idea, because it will make meta-cache hold a much larger bytes while its charge is very small...